examples/asr/notebooks/7_VAD_Offline_Online_Microphone_Demo.ipynb

```diff
@@ -0,0 +1,642 @@
+    "                    frames_per_buffer=CHUNK_SIZE) # Specifies the number of frames per buffer.\n",
+    " \n",
+    "    stream.start_stream()\n",
+    "\n",
+    "    while stream.is_active():\n",
+    "        time.sleep(0.1)\n",
+    "\n",
+    "    stream.stop_stream()\n",
+    "    stream.close()\n",
+    "    p.terminate()\n",
+    "\n",
+    "    asr.reset()\n",
+    "    return preds, proba_b, proba_s"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Here we show an example of offline streaming inference\n",
+    "You can use your file or download the provided toy dataset. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "toy_data = './vad'\n",
+    "if not os.path.exists(toy_data):\n",
+    "    !wget -c \"https://github.com/NVIDIA/NeMo/blob/master/tests/data/vad.tar.xz?raw=true\" -O vad.tar.xz \n",
+    "    !tar -xvf vad.tar.xz"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wave_file = './vad/welcome_noisy.wav'\n",
+    "CHANNELS = 1\n",
+    "RATE = 16000\n",
+    "audio, sample_rate = librosa.load(wave_file, sr=RATE)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "results = []\n",
+    "for STEP, WINDOW_SIZE in zip(STEP_LIST, WINDOW_SIZE_LIST):\n",
+    "    print(f'====== STEP is {STEP}s, WINDOW_SIZE is {WINDOW_SIZE}s ====== ')\n",
+    "    preds, proba_b, proba_s = offline_inference(wave_file, STEP, WINDOW_SIZE)\n",
+    "    results.append([STEP, WINDOW_SIZE, preds, proba_b, proba_s])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "from pylab import *\n",
+    "import numpy as np\n",
+    "import librosa.display\n",
+    "plt.figure(figsize=[16,10])\n",
+    "plt.title('Audio, Preictions and Probas')\n",
+    "plt.rcParams.update({'font.size': 10, 'font.family': 'sans-serif'})\n",
+    "subplots_adjust(hspace=2.00)\n",
+    "\n",
+    "\n",
+    "FRAME_LEN = STEP_LIST[0]\n",
+    "len_pred = len(results[0][2]) \n",
+    "\n",
+    "num = len(results)\n",
+    "for i,v in enumerate(range(num + 1)):\n",
+    "    v = v + 1\n",
+    "    if  v > len(results):\n",
+    "\n",
+    "        ax = plt.subplot(num + 2, 1, v)\n",
+    "        S = librosa.feature.melspectrogram(y=audio, sr=sample_rate, n_mels=128,\n",
+    "                                  fmax=8000)\n",
+    "        S_dB = librosa.power_to_db(S, ref=np.max)\n",
+    "        librosa.display.specshow(S_dB, x_axis='time', y_axis='mel', \n",
+    "                                 sr=sample_rate, fmax=8000)\n",
+    "        # plt.colorbar(format='%+2.0f dB')\n",
```

Comment:
Remove commented code

Reply:
Should we remove all commented code? I leave it here because it's a useful legend but it will break the alignment of the plots, so I comment it out instead deleting it in case the user wants to see it.

Reply:
Thanks for your suggestion. Fixed.